build/pack.js

```diff
@@ -63,6 +64,7 @@ module.exports = function (opts) {
     utils.collect(_c.LIB, lib);
     utils.collect(_c.DEVICES, devicesCSS, matches(".css"));
     utils.collect(_c.UI, overlays, matches("overlay.html"));
+    utils.collect(_c.UI, overlayStyles, matches("overlay.css"));
```

Comment:
Can you explain why we need this? There aren't currently any `overlay.css` files.